src/objects/BatchedMesh.js

```diff
@@ -182,6 +185,25 @@ class BatchedMesh extends Mesh {
 
 	}
 
+	_initColorsTexture() {
+
+		// layout (1 color = 1 pixel)
+		//      RGBA (=> column1)
+		//  with  8x8  pixel texture max   64 matrices * 1 pixel = (8 * 8)
+		//       16x16 pixel texture max  256 matrices * 1 pixel = (16 * 16)
+		//       32x32 pixel texture max 1024 matrices * 1 pixel = (32 * 32)
+		//       64x64 pixel texture max 4096 matrices * 1 pixel = (64 * 64)
+
+		let size = Math.sqrt( this._maxGeometryCount ); // 1 pixel needed for 1 color
+		size = Math.ceil( size );
+
+		const colorsArray = new Float32Array( size * size * 4 ); // 4 floats per RGBA pixel
```

Comment:
Since it's still one `vec4` per geometry, I'm OK with using float32 for now, to match the behavior of InstancedMesh.

Ideally, perhaps in a future PR, it might be best to use uint16. That's enough precision for Linear-sRGB or wide-gamut color spaces, and wouldn't require inline decoding in the shaders like uint8+sRGB.

Reply:
Arguably there was no way to use sRGB for the InstancedMesh color attribute and it feels odd to not set _any_ color space on the color texture. But I guess this is what would have to happen if we encoded many material properties into a single texture, anyway.

Reply:
Oh, that's a good point. My preference would be for this texture to be assigned a color space of `THREE.ColorManagement.workingColorSpace` (Linear-sRGB in 99.9% of cases). That's how we've defined `color.r` / `.g` / `.b` properties.

I wouldn't be opposed to using sRGB either, and that's certainly a change we could make internally later, but it just felt simpler to be consistent with vertex colors.

Reply:
@gkjohnson @donmccurdy Something I need to do following your discussion?

Reply:
@Kikedao  just below this line, in the DataTexture constructor, could you pass `ColorManagement.workingColorSpace` for the `colorSpace` parameter? I think that's all that is required for now.

Reply:
Done!